lib/flipper/api/v1/decorators/feature.rb

```diff
@@ -0,0 +1,33 @@
+require 'delegate'
+require 'flipper/api/v1/decorators/gate'
+
+module Flipper
+  module Api
+    module V1
+      module Decorators
+        class Feature < SimpleDelegator
+
+          # Public: The feature being decorated.
+          alias_method :feature, :__getobj__
+          
+          # Public: Returns instance as hash that is ready to be json dumped.
+          def as_json
+            {
```

Comment:
@jnunemaker totally mistake  on my end, forgot to change from when was first building it and referencing some of the ::UI responses.  We should definitely have key => key!